### PR TITLE
remote name prefix

### PR DESCRIPTION
**Pull request by geekbass**
So we can have conditional naming on remote region. Setting the default for this to 'remote' but of course could be whatever the user prefers.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -111,7 +113,7 @@ module "dcos-install" {
   public_agents_prereq-id = "${coalesce(var.public_agents_prereq-id, module.dcos-infrastructure.public_agents.prereq-id)}"
   num_public_agents       = "${coalesce(var.num_public_agents, var.num_public_agents)}"
   # DC/OS options
-  dcos_cluster_name = "${coalesce(var.dcos_cluster_name, local.cluster_name)}"
+  dcos_cluster_name                            = "${coalesce(var.dcos_cluster_name, local.cluster_name)}"
```
Comment (fatz) [inline main.tf]: this should still be `var.cluster_name` as the prefix should only influence resources but not the cluster itself

**Review by fatz — CHANGES_REQUESTED** [review]

File: main.tf [hunk 2]
```diff
@@ -83,33 +85,36 @@ module "dcos-infrastructure" {
...
 
 module "dcos-install" {
-  source = "github.com/dcos-terraform/terraform-null-dcos-install-remote-exec?ref=conditional-bootstrap"
-
   #source  = "dcos-terraform/dcos-install-remote-exec/null"
-  #version = "~> 0.0"
-
+  #version = "~> 0.1.1"
+  
+  source = "github.com/dcos-terraform/terraform-null-dcos-install-remote-exec?ref=conditional-bootstrap"
```
Comment (bernadinm) [inline main.tf]: we need to update the source back to the original

**Review by bernadinm — CHANGES_REQUESTED** [review]

File: main.tf (hunk 2 above)
Comment (fatz) [reply main.tf]: good catch indeed!

**Review by fatz — CHANGES_REQUESTED** [review]

File: main.tf (hunk 2 above)
Comment (geekbass) [reply main.tf]: Yes am aware. Was attempting a test and needed to revert to this original source. Reverting the revert now :)

**Review by geekbass — COMMENTED** [review]
@fatz @bernadinm I noticed that DC/OS module is using version 0.1.0 https://github.com/dcos-terraform/terraform-aws-dcos/blob/master/main.tf#L154

I used the lastest 0.1.1 here. Which should we specify?

**Review by fatz — APPROVED** [review]
LGTM

versions could already point to 0.2.0 but lets do it in a bulk change

**Review by bernadinm — APPROVED** [review]
LGTM

**Pull request closed without merging**